recipes/genozip/meta.yaml

```
@@ -0,0 +1,70 @@
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: genozip
+  version: 1.0.1
+
+source:
+  url: https://github.com/divonlan/genozip/raw/master/conda/genozip-1.0.1.tar.gz
```

You should use tags instead of including a tarball in the git repository:
```suggestion
  url: https://github.com/divonlan/genozip/archive/{{ version }}.tar.gz
```